### ISSUE TITLE
Issue from Doppler at #denizen-dev

### ISSUE THREAD

**Issue by spazzmatic** [DenizenScript/Denizen-For-Bukkit#338]
Skeletons spawned with the spawn command are bow-less non-shooting eunuchs who act rather like zombies, what with the arms extended and the slack-jaws...
